common/fixation_entry.go

```diff
@@ -379,7 +379,7 @@ func (fs *FixationStore) removeEntry(ctx sdk.Context, index string, block uint64
 }
 
 func (fs *FixationStore) createStoreKey(index string) string {
-	return types.EntryPrefix + fs.prefix + index
```

Comment:
how does this help us?
i feel like changing the keys is a dangerous game

Reply:
The purpose of the prefix provider by the user (fs.prefix) is to allow namespacing controlled by the user. As such, it should come *first* in the prefix/key. Otherwise, we're effectively hijacking the prefix "Entry_..." used by the fixation (and which cannot be controlled by the user). It was a mistake to set the order as it is now - it should be reversed.
There migration logic will take care of updating the store properly.